platform/node/src/node_map.cpp

```diff
@@ -278,9 +278,8 @@ void NodeMap::renderFinished() {
             [](char *, void *hint) {
                 delete reinterpret_cast<const mbgl::StillImage *>(hint);
             },
-            const_cast<mbgl::StillImage *>(img.get())
+            const_cast<mbgl::StillImage *>(img.release())
         ).ToLocalChecked();
-        img.release();
```

Comment:
Just moved it a line above so it releases directly into the `void*` in [`Nan::NewBuffer`](https://github.com/nodejs/nan/blob/master/doc/buffers.md#api_nan_new_buffer). I'm actually not sure if there's a guarantee that these arguments will be evaluated in the order I expect.